Sync `intersection-observer` from WPT upstreams

Ahmad-S792 opened the pull request:
#### 2bae6ab06c2cb5ec6564b1795b5965789e7833bc
<pre>
Sync `intersection-observer` from WPT upstreams

<a href="https://bugs.webkit.org/show_bug.cgi?id=289185">https://bugs.webkit.org/show_bug.cgi?id=289185</a>
<a href="https://rdar.apple.com/146318072">rdar://146318072</a>

Reviewed by Abrar Rahman Protyasha.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/35b7277b59398ff280b20c9f7ac5a78d3cf43771">https://github.com/web-platform-tests/wpt/commit/35b7277b59398ff280b20c9f7ac5a78d3cf43771</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/bounding-box-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/bounding-box.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/callback-cross-realm-report-exception.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/cross-origin-iframe-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/display-none.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/explicit-root-different-document.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/explicit-root-different-document.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/intersection-ratio-with-fractional-bounds-in-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/not-in-containing-block-chain.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/not-in-containing-block-chain.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/resources/intersection-observer-test-utils.js:
(checkRect):
(checkLastEntry):
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/resources/v2-subframe.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/simple-occlusion-svg-foreign-object.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/clip-path-animation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/clip-path-animation.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/clip-path-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/clip-path.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/explicit-root-different-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/explicit-root-different-document.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-child-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-child-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-iframe-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-iframe-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/grow-height-and-scrolled-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/grow-height-and-scrolled.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/not-in-containing-block-chain-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/not-in-containing-block-chain.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/padding-clip-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/padding-clip.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/resources/v2-midframe.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/root-vertical-rl-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/root-vertical-rl.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-and-root-margin-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-and-root-margin.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-4-val-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-4-val.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-clip-path-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-clip-path.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-dynamic-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-dynamic.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-horizontal-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-horizontal.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-iframe-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested-2-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested-2.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested-3-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested-3.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-no-intersect-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-no-intersect.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-non-scrolling-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-non-scrolling-root.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-not-contained-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-not-contained.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-percent-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-percent.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-with-border-outline-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-with-border-outline.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-zero-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-zero.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-clipped-rect-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-clipped-rect-target.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-container-element-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-container-element.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-group-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-group-target.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-image-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-image.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-intersection-with-fractional-bounds-2-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-intersection-with-fractional-bounds-2.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-intersection-with-fractional-bounds-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-intersection-with-fractional-bounds.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-rect-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-rect-target.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-stroke-change-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-stroke-change.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-target-changes-position-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-target-changes-position.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-transformed-rect-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-transformed-rect-target.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-viewbox-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-viewbox.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/box-reflect-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/box-reflect.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/nested-cross-origin.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/nested-cross-origin.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/visibility-hidden-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/visibility-hidden.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/zoom-scaled-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/zoom-scaled-target.html:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/intersection-observer/svg-clipped-rect-target-expected.txt: Platform Specific Expectation
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/intersection-observer/svg-intersection-with-fractional-bounds-expected.txt: Ditto
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/intersection-observer/svg-rect-target-expected.txt: Ditto
* LayoutTests/platform/ios/TestExpectations: Skip failing and flaky tests

Canonical link: <a href="https://commits.webkit.org/291758@main">https://commits.webkit.org/291758@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/2da3e2dad43840423c2cd3373479d762f021a234

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/93937 "Passed style check") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/13522 "Build is in progress. Recent messages:OS: Sequoia (15.2), Xcode: 16.2; Skipping applying patch since patch_id isn't provided; Checked out pull request; Compiled WebKit (warnings)") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/3257 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/98944 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/44464 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/13819 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/21953 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/71677 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/29028 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/96939 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/132/builds/10257 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/84853 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/52015 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/9941 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/2498 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/43780 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/80200 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/2580 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/100983 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/20989 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/15298 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/80694 "Build is in progress. Recent messages:") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/21241 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/80822 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/80049 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/19925 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/24596 "Passed tests") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/1948 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/14146 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/20973 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/20660 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/24120 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/22401 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aprotyas commented on LayoutTests/TestExpectations:
```diff
@@ -4780,6 +4780,7 @@ webkit.org/b/230593 fast/block/positioning/056.html [ Failure ]
 # wpt intersection-observer failures
 webkit.org/b/208052 imported/w3c/web-platform-tests/intersection-observer/v2/cross-origin-effects.sub.html [ Skip ]
 webkit.org/b/208052 imported/w3c/web-platform-tests/intersection-observer/v2/cross-origin-occlusion.sub.html [ Skip ]
+imported/w3c/web-platform-tests/intersection-observer/v2/nested-cross-origin.sub.html [ Skip ]
```
Is this a timeout? Consider filing a bug if so

Ahmad-S792 replied:
Yes - it is timeout. I kept expectations file, so it is easier to update, when we do `--reset-results`.

Ahmad-S792 replied:
Also will file bug and will update this once EWS finishes to see, if I need to do anything else here and there.

Ahmad-S792 replied:
@aprotyas - done - all times out were tracked in single bug and then I noticed all of them are v2 related. So I added link to v2 support bug as well.

Ahmad-S792 committed:
Sync `intersection-observer` from WPT upstreams

https://bugs.webkit.org/show_bug.cgi?id=289185
rdar://146318072

Reviewed by Abrar Rahman Protyasha.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/35b7277b59398ff280b20c9f7ac5a78d3cf43771

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/bounding-box-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/bounding-box.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/callback-cross-realm-report-exception.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/cross-origin-iframe-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/display-none.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/explicit-root-different-document.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/explicit-root-different-document.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/intersection-ratio-with-fractional-bounds-in-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/not-in-containing-block-chain.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/not-in-containing-block-chain.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/resources/intersection-observer-test-utils.js:
(checkRect):
(checkLastEntry):
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/resources/v2-subframe.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/simple-occlusion-svg-foreign-object.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/clip-path-animation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/clip-path-animation.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/clip-path-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/clip-path.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/explicit-root-different-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/explicit-root-different-document.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-child-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-child-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-iframe-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-iframe-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-scroll-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/fixed-position-scroll.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/grow-height-and-scrolled-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/grow-height-and-scrolled.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/not-in-containing-block-chain-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/not-in-containing-block-chain.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/padding-clip-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/padding-clip.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/resources/v2-midframe.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/root-vertical-rl-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/root-vertical-rl.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-and-root-margin-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-and-root-margin.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-4-val-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-4-val.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-clip-path-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-clip-path.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-dynamic-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-dynamic.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-horizontal-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-horizontal.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-iframe-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested-2-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested-2.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested-3-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested-3.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-nested.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-no-intersect-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-no-intersect.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-non-scrolling-root-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-non-scrolling-root.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-not-contained-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-not-contained.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-percent-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-percent.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-with-border-outline-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-with-border-outline.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-zero-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin-zero.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/scroll-margin.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-clipped-rect-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-clipped-rect-target.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-container-element-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-container-element.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-group-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-group-target.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-image-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-image.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-intersection-with-fractional-bounds-2-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-intersection-with-fractional-bounds-2.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-intersection-with-fractional-bounds-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-intersection-with-fractional-bounds.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-rect-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-rect-target.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-stroke-change-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-stroke-change.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-target-changes-position-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-target-changes-position.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-transformed-rect-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-transformed-rect-target.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-viewbox-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/svg-viewbox.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/box-reflect-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/box-reflect.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/nested-cross-origin.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/nested-cross-origin.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/v2/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/visibility-hidden-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/visibility-hidden.html:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/zoom-scaled-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/intersection-observer/zoom-scaled-target.html:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/intersection-observer/svg-clipped-rect-target-expected.txt: Platform Specific Expectation
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/intersection-observer/svg-intersection-with-fractional-bounds-expected.txt: Ditto
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/intersection-observer/svg-rect-target-expected.txt: Ditto
* LayoutTests/platform/ios/TestExpectations: Skip failing and flaky tests

Canonical link: https://commits.webkit.org/291758@main
